frontend/src/routes/Governance/common/util.tsx

```diff
@@ -492,19 +494,61 @@ export function getPolicyRemediation(policy: Policy | undefined) {
   }
 
   remediationAggregation = remediationAggregation.trim().replaceAll(' ', '/')
-  if (remediationAggregation === '') {
-    return '-'
+
+  if (rootRA === REMEDIATION_ACTION.ENFORCE) {
+    return remediationAggregation.includes(REMEDIATION_ACTION.INFORM_ONLY) ? rootRA.concat('/informOnly') : rootRA
   }
-  if (policy.spec?.remediationAction) {
-    if (remediationAggregation.includes('informOnly')) {
-      if (remediationSet.size > 1) {
-        return policy.spec.remediationAction.concat('/informOnly')
-      }
-      return 'informOnly'
+
+  if (rootRA) {
+    if (remediationAggregation.includes(REMEDIATION_ACTION.INFORM_ONLY)) {
+      rootRA = remediationSet.size > 1 ? rootRA.concat('/informOnly') : REMEDIATION_ACTION.INFORM_ONLY
+    }
+  } else {
+    rootRA = remediationAggregation
+    // empty root remediation and template remedation all empty
+    if (!rootRA) {
+      return '-'
+    }
+  }
+
+  let remediationOverride = ''
+  let allOverridden = propagatedPolicies.length !== 0
+  propagatedPolicies.forEach((propaPolicy) => {
+    const propaPolicyRA = propaPolicy.spec.remediationAction
+    if (!propaPolicyRA) {
+      allOverridden = false
+      return
     }
-    return policy.spec.remediationAction
+    if (propaPolicyRA === rootRA.replace('/informOnly', '')) {
+      allOverridden = false
+      return
+    }
+
+    if (!remediationOverride) {
+      remediationOverride = REMEDIATION_ACTION.ENFORCE
+    }
+  })
+
+  if (allOverridden) {
+    return rootRA.includes('informOnly')
+      ? REMEDIATION_ACTION.INFORMONLY_ENFORCE_OVERRIDDEN
+      : REMEDIATION_ACTION.ENFORCE_OVERRIDDEN
   }
-  return remediationAggregation
+
+  // No overrides are present
+  if (!remediationOverride) {
+    return rootRA
+  }
+
+  return rootRA.includes('informOnly')
+    ? REMEDIATION_ACTION.INFORM_INFORMONLY_ENFORCE_OVERRIDDEN
+    : REMEDIATION_ACTION.INFORM_ENFORCE_OVERRIDDEN
+}
+
+export function getPolicyTempRemediation(propagatedPolicy: Policy, template: PolicyTemplate | undefined): string {
+  const propagatedRA = propagatedPolicy.spec.remediationAction
+  const objectRemediation = template?.objectDefinition.spec.remediationAction
+  return propagatedRA || objectRemediation || ''
```

Comment:
This will need to change to account for `informOnly`.

Reply:
you mean adding comment?

Reply:
It was my mistake. This is right.